apps/directory/src/stores/filtersStore.ts

```diff
@@ -338,8 +338,12 @@ export const useFiltersStore = defineStore("filtersStore", () => {
     bookmarkTriggeredFilter.value = fromBookmark ?? false;
 
     if (typeof value === "string" || typeof value === "boolean") {
-      filters.value[filterName] = value;
-      checkoutStore.setSearchHistory(`${filterName} filtered on ${value}`);
+      if (value === "") {
+        delete filters.value[filterName];
```

Comment:
Shouldn't you update the search history to say that the filter was removed (like it is done for arrays)?

Reply:
Right you are! Added it.